content/applications/marketing/marketing_automation/target_audience.rst

```diff
@@ -1,123 +1,123 @@
-the :guilabel:`Target` field, and click :guilabel:`Search More` from the drop-down menu. Then,
-search for :guilabel:`Lead/Opportunity`, and select it.
-
-Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Then, click on the default
-:guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a
-drop-down menu full of filter options. From this drop-down, scroll down (or search for)
-:guilabel:`Type`.
-
-Keep the second portion of the filter equation on the default :guilabel:`🟰 (equal sign)` icon.
-
-Next, change the third (and final) portion of the filter equation from :guilabel:`Lead` to
-:guilabel:`Opportunity`. The number of :guilabel:`Records` that fit this specific filter equation
-changes as the equation is customized.
-
-Add another node to this filter by clicking the :guilabel:`➕ (plus sign)` icon to the right of the
-equation.
-
-With "new" opportunities being the target of this filter, the second node will focus on *only*
-locating opportunities that are in the :guilabel:`New` stage of the pipeline. To do that, select
-the default :guilabel:`ID` from the first portion of the second filter equation, and scroll down (or
-search for) :guilabel:`Stage` from the field drop-down menu.
-
-Once again, leave the second portion of the filter equation on :guilabel:`🟰 (equal sign)` icon.
-
-Lastly, highlight the default value in the third (and final) portion of the second filter equation,
-and type in `New`. With that in place, Odoo only targets opportunities that are in the "New" stage
-of the pipeline.
-
-.. image:: target_audience/filters-opportunities.png
-   :align: center
-   :alt: A standard scenario using filters in the Odoo Marketing Automation app.
-
-Scenario #2: Narrow down target to event attendees who purchased a specific ticket
-----------------------------------------------------------------------------------
-
-While in *Edit mode* on a campaign template form (by clicking the :guilabel:`Edit` button), select
-the :guilabel:`Target` field, and click :guilabel:`Search More` from the drop-down menu. Then,
-scroll down (or search for) :guilabel:`Event`, and select it.
-
-Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default
-:guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a
-drop-down menu full of filter options. From this drop-down, scroll down (or search for)
-:guilabel:`Event`.
-
-Click the default :guilabel:`🟰 (equal sign)` icon in the second portion of the filter equation. This
-reveals a drop-down menu. From this drop-down menu, select :guilabel:`contains`.
+- :guilabel:`Target`: defines the model of the target audience for the campaign. The assigned model
+  determines the fields that are available throughout the campaign, including the fields available
+  in the :guilabel:`Filter` section, and in dynamic placeholders.
```

Comment:
- would add guis here since we're doing form coverage
- take earlier definition we removed and put it here since it's more digestible and easier to understand for folks

```suggestion
- :guilabel:`Target`: specifies the type of records available for use in the campaign, such as
  :guilabel:`Lead/Opportunity`, :guilabel:`Event Registration`, :guilabel:`Contact`, The assigned
  records model determines the fields that are available throughout the campaign, including the
  fields available in the :guilabel:`Filter` section, and in dynamic placeholders.
```